chains/orchestrator-relays/runtime/starlight/src/lib.rs

```diff
@@ -0,0 +1,3813 @@
+        }
+    }
+}
+
+parameter_types! {
+    pub const OnDemandTrafficDefaultValue: FixedU128 = FixedU128::from_u32(1);
+    // Keep 2 blocks worth of revenue information.
+    // We don't need this because it is only used by coretime and we don't have coretime,
+    // but the pallet implicitly assumes that this bound is at least 1, so we use a low value
+    // that won't cause problems.
+    pub const MaxHistoricalRevenue: BlockNumber = 2;
+    pub const OnDemandPalletId: PalletId = PalletId(*b"py/ondmd");
+}
+
+impl parachains_assigner_on_demand::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type Currency = Balances;
+    type TrafficDefaultValue = OnDemandTrafficDefaultValue;
+    type WeightInfo = weights::runtime_parachains_assigner_on_demand::SubstrateWeight<Runtime>;
+    type MaxHistoricalRevenue = MaxHistoricalRevenue;
+    type PalletId = OnDemandPalletId;
+}
+
+impl parachains_initializer::Config for Runtime {
+    type Randomness = pallet_babe::RandomnessFromOneEpochAgo<Runtime>;
+    type ForceOrigin = EnsureRoot<AccountId>;
+    type WeightInfo = weights::runtime_parachains_initializer::SubstrateWeight<Runtime>;
+    type CoretimeOnNewSession = ();
+}
+
+impl parachains_disputes::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type RewardValidators = RewardValidators;
+    type SlashingHandler = parachains_slashing::SlashValidatorsForDisputes<ParasSlashing>;
+    type WeightInfo = weights::runtime_parachains_disputes::SubstrateWeight<Runtime>;
+}
+
+impl parachains_slashing::Config for Runtime {
+    type KeyOwnerProofSystem = Historical;
+    type KeyOwnerProof =
+        <Self::KeyOwnerProofSystem as KeyOwnerProofSystem<(KeyTypeId, ValidatorId)>>::Proof;
+    type KeyOwnerIdentification = <Self::KeyOwnerProofSystem as KeyOwnerProofSystem<(
+        KeyTypeId,
+        ValidatorId,
+    )>>::IdentificationTuple;
+    type HandleReports = parachains_slashing::SlashingReportHandler<
+        Self::KeyOwnerIdentification,
+        Offences,
+        ReportLongevity,
+    >;
+    type WeightInfo = parachains_slashing::TestWeightInfo;
+    type BenchmarkingConfig = parachains_slashing::BenchConfig<200>;
+}
+
+parameter_types! {
+    pub const ParaDeposit: Balance = 40 * UNITS;
+}
+
+impl paras_registrar::Config for Runtime {
+    type RuntimeOrigin = RuntimeOrigin;
+    type RuntimeEvent = RuntimeEvent;
+    type Currency = Balances;
+    type OnSwap = ();
+    type ParaDeposit = ParaDeposit;
+    type DataDepositPerByte = DataDepositPerByte;
+    type WeightInfo = weights::runtime_common_paras_registrar::SubstrateWeight<Runtime>;
+}
+
+impl pallet_parameters::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type RuntimeParameters = RuntimeParameters;
+    type AdminOrigin = DynamicParameterOrigin;
+    type WeightInfo = weights::pallet_parameters::SubstrateWeight<Runtime>;
+}
+
+parameter_types! {
+    // TODO: BondingDuration is set to 28 days on Polkadot,
```

Comment:
you can remove this TODO, I think 28 is a safe compromise

Reply:
fixed